api/node/consts.js

```diff
@@ -0,0 +1,6 @@
+// const name by W4nder
+const STEAM_USER_HIGH_VALUE = BigInt(76561197960265728)
+
+module.exports = {
+    STEAM_USER_HIGH_VALUE
+}
```

Comment:
Is it only for the internal API usage? Because we already have constants.js file where you can store your consts

Reply:
oh, I miss this one, it could be moved to this file.